### PR TITLE
Feat create and devops rules

### PR DESCRIPTION
**Pull request by Dldvlpr**
FEAT !:
-Method addProject.
-ProjectType.
-Add constraints in entity.
-Add toast component for notify when the project is created.
-Template/new.index.html.
- Twig rules for use bootstrap form.

-Add rules for phpcs.xml.dist & phpmd.xml.
-Modify grumphp.yml for use right files.

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -10,10 +10,18 @@
 
     <rule ref="PSR12"/>
 
-    <file>bin/</file>
+    <rule ref="Generic.Files.LineEndings.InvalidEOLChar">
+        <message>End of line character is invalid.
+Dear Wilder using Windows, it seems you miss something in readme during project install. You should edit your IDE settings as recommended in the install instruction.</message>
```
Comment (aldaitzwild) [inline phpcs.xml.dist]: I'm ok to add the rule but the message is not relevant

File: phpmd.xml [hunk 2]
```diff
@@ -1,24 +1,22 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<ruleset name="PHPMD rule set"
+<?xml version="1.0"?>
+<ruleset name="My first PHPMD rule set"
```
Comment (aldaitzwild) [inline phpmd.xml]: Find a better name : "PRTracker rule set" or something like that

File: phpmd.xml [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
-    http://pmd.sf.net/ruleset_xml_schema.xsd"
->
-    <rule ref="rulesets/cleancode.xml" >
-        <exclude name="ElseExpression" />
-        <exclude name="StaticAccess" />
-        <exclude name="BooleanArgumentFlag" />
-    </rule>
+                       http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>
+        My custom rule for PR tracker
```
Comment (aldaitzwild) [inline phpmd.xml]: ... with a real description

File: src/Controller/ProjectController.php [hunk 4]
```diff
@@ -19,4 +23,27 @@ public function index(ProjectRepository $projectRepository): Response
             'projects' => $projects,
         ]);
     }
+
+    #[Route('/addProject', name: 'add')]
+    public function addProject(EntityManagerInterface $entityManager, Request $request): Response
```
Comment (aldaitzwild) [inline src/Controller/ProjectController.php]: No you shouldn't use the EntityManager here, but the Repository to save your object

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: phpmd.xml (hunk 2 above)
Comment (Dldvlpr) [reply phpmd.xml]: add perfect title

File: phpmd.xml (hunk 3 above)
Comment (Dldvlpr) [reply phpmd.xml]: real description for all rules and exclude

File: src/Controller/ProjectController.php (hunk 4 above)
Comment (Dldvlpr) [reply src/Controller/ProjectController.php]: change by repository and tested

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**